src/opnsense/service/templates/OPNsense/Monit/monitrc

```diff
@@ -164,3 +164,5 @@ check {{ service.type }} {{ service.name }} {{ path }}
 {%   endfor %}
 {%  endif %}
 {% endif %}
+
+include /usr/local/etc/monit.conf.d/*.conf
```

Comment:
In https://github.com/opnsense/docs/pull/455/files you added "/usr/local/etc/monit.opnsense.d" which I would recommend in  case monit starts using monit.conf.d by default which could clash with user-based configuration

Reply:
Thanks for reviewing it so fast – I will add the missing files and make sure the paths are aligned 👍